.github/workflows/build-mudlet.yml

```diff
@@ -64,7 +64,13 @@ jobs:
     - name: (Linux) Install apt dependencies
       if: runner.os == 'Linux'
       run: |
-        # these aren't available or don't work well in vcpkg
+        sudo apt update
+        # The above is enough to tell the OS's package system what versions
```

Comment:
How about:

> fix issues trying to install outdated packages

Reply:
Well that wouldn't have been enough for me to come rapidly to the conclusion that both an update and THEN an *upgrade* was NOT needed...

Reply:
No... you thought both were needed, but I realised it wasn't :)

I'd prefer shorter comments, rather than long verbose ones that take cognitive load to read. Add them everywhere in the code and you spend more time reading comments than the actual code... some code necessicates them, but not this simple fix.